chart/values.schema.json

```diff
@@ -54,46 +54,13 @@
                   "https"
                 ]
               },
-              "apiValidateTls": {
-                "type": "boolean"
-              },
```

Comment:
Please don't delete the JSON Schema entry for the `apiValidateTls` helm value if it stays